packages/inference/src/tasks/nlp/chatCompletion.ts

```diff
@@ -0,0 +1,31 @@
+import { InferenceOutputError } from "../../lib/InferenceOutputError";
+import type { BaseArgs, Options } from "../../types";
+import { request } from "../custom/request";
+import type { ChatCompletionInput, ChatCompletionOutput } from "@huggingface/tasks";
```

Comment:
This will cause issues when importing the lib - see https://github.com/huggingface/huggingface.js/issues/584

We either need to solve https://github.com/huggingface/huggingface.js/issues/584 not import the types from another package

Reply:
yes, maybe we could copy the types for now, since this method is need soon? 
we only need `ChatCompletionInput, ChatCompletionStreamOutput, ChatCompletionOutput` wdyt?

Reply:
I'll look at it today

Reply:
https://github.com/huggingface/huggingface.js/pull/651